### ISSUE TITLE
`formatjson` throws an error when used with `print teamdrives`

### ISSUE THREAD

**Issue by Prodeguerriero** [taers232c/GAMADV-XTD3#409]
Hi there,

Just a quick one: if I run `gam user me@mydomain.com print teamdrives formatjson` I get the following error message:
```python
Traceback (most recent call last):
  File "__init__.py", line 74485, in ProcessGAMCommand
  File "__init__.py", line 63038, in printShowSharedDrives
  File "__init__.py", line 8835, in writeCSVfile
  File "__init__.py", line 8461, in writeCSVToStdout
  File "__init__.py", line 8429, in writeCSVData
  File "csv.py", line 167, in writerows
  File "csv.py", line 159, in _dict_to_list
ValueError: dict contains fields not in fieldnames: 'role'
```

Cheers,

**Comment by taers232c** [comment]
It's  bug, I will fix.

Ross
----
Ross Scroggs
***@***.***

**Comment by taers232c** [comment]
Fixed 6.74.02
----
Ross Scroggs
***@***.***
